cli/lib/kontena/cli/registry/create_command.rb

```diff
@@ -87,6 +87,11 @@ def execute
 
       data = {
         name: 'registry',
+        stack: 'kontena/registry',
+        version: Kontena::Cli::VERSION,
+        source: '---',
+        registry: 'file://',
```

Comment:
Actually load the YAML from some CLI resource file?

Use something magical like `internal://registry`?

File an issue for moving these to the stack registry, so we don't forget?